adyenv6core/src/com/adyen/v6/factory/AdyenRequestFactory.java

```diff
@@ -960,6 +964,14 @@ private void setSepaDirectDebitData(PaymentsRequest paymentRequest, CartData car
         paymentRequest.setPaymentMethod(paymentMethodDetails);
     }
 
+    private void setPixData(PaymentsRequest paymentsRequest, CartData cartData) {
+        Name shopperName = new Name();
+        shopperName.setFirstName(cartData.getAdyenFirstName());
+        shopperName.setLastName(cartData.getAdyenLastName());
+        paymentsRequest.setShopperName(shopperName);
+        paymentsRequest.setSocialSecurityNumber(cartData.getAdyenSocialSecurityNumber());
+    }
```

Comment:
do we want/need to send lineitems here? https://docs.adyen.com/payment-methods/pix/web-component#make-a-payment

Reply:
Im not sure if we want to do it or not. Because as per code we pass it only for open invoice payment methods such as Klarna, AfterPay etc.

Reply:
updated it.